src/styles/index.scss

```diff
@@ -1,3 +1,82 @@
+* {
+  box-sizing: border-box;
+}
+
 body {
   margin: 0;
+  display: flex;
+  align-items: center;
+  justify-content: center;
+  height: 100vh;
+}
+
+$color-gray: #eee;
+$color-border: #000;
+$color-hover: #ffbfcb;
+$cell-size: 100px;
+$border-size: 1px;
+$font-size: 30px;
+$hover-transform: -20px;
+$animation-duration: 0.5s;
+$padding: 10px;
```

Comment:
It is better to place a large number of variables in a separate ``variable.scss`` file.